libs/datamodel/core/src/transform/attributes/unique_and_index.rs

```diff
@@ -146,8 +146,8 @@ trait IndexAttributeBase<T>: AttributeValidator<T> {
             .default_arg("fields")?
             .as_array()
             .iter()
-            .map(|f| f.as_constant_literal().unwrap())
```

Comment:
Can you do the same fix for @@id as well? I checked there we also unwrap

Reply:
Apart from this 👍

Reply:
Good catch! Will add as an extra commit in this PR.

Reply:
Great!